remind.py

```diff
@@ -0,0 +1,119 @@
+from functions import generate_html_file, get_osp_version, has_blockers
+
+import jinja2
+from smtplib import SMTP
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+
+
+def run_remind(config, blockers, server, header):
+
+	# get set of all owners in blocker file
+	owner_list = []
+	for job in blockers:
+		owners = blockers[job].get('owners', False)
+		if not owners:
+			continue
+		owner_list.extend(owners)
+	owner_set = set(owner_list)
+
+	# find each job with no blockers including the owner and send email with agg'd list
+	for owner in owner_set:
+		rows = []
+		for job_name in blockers:
+			owners = blockers[job_name].get('owners', [])
+			if owner not in owners:
+				continue
+			else:
```

Comment:
This `else` clause looks to be redundant.

Reply:
Yup, will remove in next changeset